BrightID/src/components/Onboarding/ImportFlow/thunks/channelUploadThunks.ts

```diff
@@ -13,7 +13,7 @@ import {
 } from '@/utils/channels';
 import { IMPORT_PREFIX, RECOVERY_CHANNEL_TTL } from '@/utils/constants';
 
-export const uploadAllInfoAfter = async (after) => {
+export const getUserInfo = async () => {
   const {
     user,
     keypair: { publicKey: signingKey },
```

Comment:
There are many unused variables. It looks like only `user` is actually used below.

Reply:
This is because our goal is to use this local server for the import flow, and this part of the code is refactored to be used both in the ChannelApi and the LocalServer methods. I'm currently just giving the password and explorer code for the beginning because we currently need them in Aura

Reply:
You're right. fixed